React/wbooks/src/Components/FormInput.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
In components, directories and files should have the same structure as screens. So here you should have a FormInput directory, with an index.js file with this component and a styles.scss with the style (instead of the forms.scss)